_posts/2018-03-18-phoenix-live-view.md

```diff
@@ -0,0 +1,341 @@
+---
+author: Sophie DeBenedetto
+author_link: https://github.com/sophiedebenedetto
+categories: til
+date: 2018-12-17
+layout: post
+title:  Walk-Through of Phoenix Live View
+excerpt: >
+  Learn how to use Phoenix Live View for real-time features without complicated JS frameworks.
+---
+It's here! Phoenix Live View leverages server-rendered HTML and Phoenix's native WebSocket tooling so you can build fancy real-time features without all that complicated JavaScript. If you're sick to death of writing JS (I had a bad day with Redux, don't ask), then this is the library for you!
+
+Phoenix Live View is brand brand new so I thought I'd provide short write-up of a super simple demo I built for anyone looking to get up and running.
```

Comment:
Should we mention that it's so new, it's still a release candidate and subject to changes?

Reply:
"so I thought I'd provide a short write-up of" — you're missing "a" in your original